containers/HomeHeaderRight/HomeHeaderRight.js

```diff
@@ -5,6 +5,7 @@ import React, { PureComponent } from 'react';
 import { View, Text, StyleSheet, TouchableOpacity } from 'react-native';
 import { connect } from 'react-redux';
 import { showAccountIssues } from 'actions';
+import I18n from 'utils';
```

Comment:
Incredibly, earned somehow! More precisely, I tried to import via `import I18n from 'utils / i18n'` although I did so early, but it did not work. In short, hallelujah, it works, although I feel dumb,  because again I do not understand WHY.